packages/mobile/e2e/src/usecases/NewAccountOnboarding.js

```diff
@@ -25,11 +35,44 @@ export default NewAccountOnboarding = () => {
     await enterPinUi()
     await enterPinUi()
 
+    // Dismiss banners if present
+    await dismissBanners()
+
     // Skip Phone Number verification
     await element(by.id('VerificationEducationSkipHeader')).tap()
     await element(by.id('VerificationSkipDialog/PrimaryAction')).tap()
 
     // Arrived to Home screen
     await expect(element(by.id('SendOrRequestBar'))).toBeVisible()
   })
+
+  // Ideally this wouldn't be dependent on the previous test
+  // Skip setup on android for now
+  if (device.getPlatform() === 'ios') {
```

Comment:
yea this is not great if we shuffle the tests around or try to reuse. Why did we need this again?

Reply:
This was because currently `.getAttributes()`, needed to parse the account key, isn't working on Android for Detox;  however, a recent update should have enabled it: types just need to be [updated in Detox](https://github.com/wix/Detox/pull/2771).